README.md

```diff
@@ -63,18 +67,18 @@ The Flow model is aligned with the principles and schemas of [AMWA NMOS IS-04](h
 The process of reading from the store is:
 
 1. Client identifies the Flow ID and timerange of interest
-2. Client makes a request to [`GET flows/<flow_id>/segments?timerange=<timerange>`](https://bbc.github.io/tams/#/operations/GET_flows-flowId-segments) and receives a list of segments, timeranges and download URLs
-3. Client downloads each URL, concatenates the segments together and unwraps the grains within
+2. Client makes a request to [`GET flows/<flow_id>/segments?timerange=<timerange>`](https://bbc.github.io/tams/#/operations/GET_flows-flowId-segments) and receives a list of Flow Segments, including their timeranges and download URLs
+3. Client downloads each URL, concatenates the Flow Segments together and unwraps the grains within
 4. The first and last Flow Segment may contain more grains than requested, so the client should skip any received not in the requested timerange
 
 The process of writing to the store is:
 
 1. Client creates a Flow if necessary by making a request to [`PUT flows/<flow_id>`](https://bbc.github.io/tams/#/operations/PUT_flows-flowId)
 2. Client makes a request to [`POST flows/<flow_id>/storage`](https://bbc.github.io/tams/#/operations/POST_flows-flowId-storage) with the timerange to be written
-3. Store responds with a list of segment timeranges and URLs to PUT segments to, along with an optional `pre` URL to call before writing
+3. Store responds with a list of Flow Segment timeranges and URLs to PUT media data into, along with an optional `pre` URL to call before writing
```

Comment:
This is OK for now, but will need revisiting after Sam has made his diagrams about Segments, and we've properly shaken down the questions around fixed-size segments. I'll pop in another ticket for that.